distro/configuration/ampathforms/hiv_care_follow_up_form.json

```diff
@@ -259,6 +259,353 @@
                 }
               ]
             },
+            {
+              "label": "Current cough",
+              "type": "obs",
+              "id": "cOughDuratiOn",
```

Comment:
@Michaelndula please make the IDs consistent by maintaining a camelcase naming convention for the Ids